.readthedocs.yaml

```diff
@@ -1,9 +1,19 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
 python:
-  version: 3.7
+  version: 3.11
```

Comment:
This should be removed - it's been deprecated. The version is only needed in `build.tools.python`.
https://docs.readthedocs.io/en/stable/config-file/v2.html#python-version

Reply:
Good catch :) 

I mistakenly thought the version was needed for the install step to function.